crates/mysten-common/src/util.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::in_antithesis;
+use crate::random::get_rng;
+use rand::seq::SliceRandom;
+use rand::Rng;
+
+pub fn randomize_cache_capacity<T>(size: T) -> T
```

Comment:
minor: since this is written for release mode code and runs in release mode as well, how about calling it `randomize_cache_capacity_in_test()` or just `cache_capacity()`? Otherwise seeing `LruCache::new(
                NonZeroUsize::new(randomize_cache_capacity(PROCESSED_CACHE_CAP)).unwrap()` may make people think the capacity is randomized in release mode too.